src/meta/api/endpoint/ApiEndPoint.ts

```diff
@@ -62,7 +62,10 @@ export const ApiEndPoint = {
     },
 
     Repository: {
-      one: () => apiPath('cycle-data', 'repository'),
+      one: () => apiPath('cycle-data', 'repository', 'repository'),
+      file: (uuid = ':uuid') => apiPath('cycle-data', 'repository', 'file', uuid),
```

Comment:
We use it in filePath to avoid passing query params in links, should we instead add query params?
We used this convention in previous implementations, eg. 
`      one: (uuid = ':uuid') => apiPath('file', 'assessment', uuid),`